scripts/globals/ability.lua

```diff
@@ -10,424 +10,620 @@ tpz = tpz or {}
+    ACCESSION          = 218,
+    PARSIMONY          = 219,
+    ALACRITY           = 220,
+    EBULLIENCE         = 221,
+    MANIFESTATION      = 222,
+    STRATAGEMS         = 223,
+    VELOCITY_SHOT      = 224,
+    SNARL              = 225,
+    RETALIATION        = 226,
+    FOOTWORK           = 227,
+    DESPOIL            = 228,
+    PIANISSIMO         = 229,
+    SEKKANOKI          = 230,
+    ELEMENTAL_SIPHON   = 232,
+    SUBLIMATION        = 233,
+    ADDENDUM_WHITE     = 234,
+    ADDENDUM_BLACK     = 235,
+    COLLABORATOR       = 236,
+    SABER_DANCE        = 237,
+    FAN_DANCE          = 238,
+    NO_FOOT_RISE       = 239,
+    ALTRUISM           = 240,
+    FOCALIZATION       = 241,
+    TRANQUILITY        = 242,
+    EQUANIMITY         = 243,
+    ENLIGHTENMENT      = 244,
+    AFFLATUS_SOLACE    = 245,
+    AFFLATUS_MISERY    = 246,
+    COMPOSURE          = 247,
+    YONIN              = 248,
+    INNIN              = 249,
+    AVATARS_FAVOR      = 250,
+    READY              = 251,
+    RESTRAINT          = 252,
+    PERFECT_COUNTER    = 253,
+    MANA_WALL          = 254,
+    DIVINE_EMBLEM      = 255,
+    NETHER_VOID        = 256,
+    DOUBLE_SHOT        = 257,
+    SENGIKORI          = 258,
+    FUTAE              = 259,
+    SPIRIT_JUMP        = 260,
+    PRESTO             = 261,
+    DIVINE_WALTZ_II    = 262,
+    FLOURISHES_III     = 263,
+    CLIMACTIC_FLOURISH = 264,
+    LIBRA              = 265,
+    TACTICAL_SWITCH    = 266,
+    BLOOD_RAGE         = 267,
+    IMPETUS            = 269,
+    DIVINE_CARESS      = 270,
+    SANCROSANCTITY     = 271,
+    ENMITY_DOUSE       = 272,
+    MANAWELL           = 273,
+    SABOTEUR           = 274,
+    SPONTANEITY        = 275,
+    CONSPIRATOR        = 276,
+    SEPULCHER          = 277,
+    PALISADE           = 278,
+    ARCANE_CREST       = 279,
+    SCARLET_DELIRIUM   = 280,
+    SPUR               = 281,
+    RUN_WILD           = 282,
+    TENUTO             = 283,
+    MARCATO            = 284,
+    BOUNTY_SHOT        = 285,
+    DECOY_SHOT         = 286,
+    HAMANOHA           = 287,
+    HAGAKURE           = 288,
+    ISSEKIGAN          = 291,
+    DRAGON_BREAKER     = 292,
+    SOUL_JUMP          = 293,
+    STEADY_WING        = 295,
+    MANA_CEDE          = 296,
+    EFFLUX             = 297,
+    UNBRIDLED_LEARNING = 298,
+    TRIPLE_SHOT        = 301,
+    ALLIES_ROLL        = 302,
+    MISER_S_ROLL       = 303,
+    COMPANION_S_ROLL   = 304,
+    AVENGER_S_ROLL     = 305,
+    COOLDOWN           = 309,
+    DEUX_EX_AUTOMATA   = 310,
+    CURING_WALTZ_V     = 311,
+    FEATHER_STEP       = 312,
+    STRIKING_FLOURISH  = 313,
+    TERNARY_FLOURISH   = 314,
+    PERPETUANCE        = 316,
+    IMMANENCE          = 317,
+    SMITING_BREATH     = 318,
+    RESTORING_BREATH   = 319,
+    KONZEN_ITTAI       = 320,
+    BULLY              = 321,
+    MAINTENANCE        = 322,
+	BRAZEN_RUSH        = 323,
+	INNER_STRENGTH     = 324,
+	ASYLUM             = 325,
```

Comment:
Seems to be tab characters throughout these lists~

Reply:
It is working this way. I used tab on the abilities that needed to be programmed, I’ll remove them if needed just let me know. Thanks!

Reply:
took out tab keys